src/functions-reference/array_operations.Rmd

```diff
@@ -409,3 +410,21 @@ Number of components of v less than v[s]
 `int` **`rank`**`(int[] v, int s)`<br>\newline
 Number of components of v less than v[s]
 
+## Reversing Functions {#reversing-functions}
+
+Reversing allows to create a copy of array in which elements are in reverse
```

Comment:
This isn't quite grammatical and it's not a copy.  Also, we want to make the doc more active.  So this should be:

```
Stan provides functions to create a new array by reversing the order of elements in an existing array.  For example, ...
```

Reply:
Thanks for the much improved wording!